agent/xds/config.go

```diff
@@ -168,5 +168,12 @@ func ToOutlierDetection(p *structs.PassiveHealthCheck) *envoy_cluster_v3.Outlier
 	if p.MaxFailures != 0 {
 		od.Consecutive_5Xx = &wrappers.UInt32Value{Value: p.MaxFailures}
 	}
+
+	if p.EnforcingConsecutive5xx != nil {
+		od.EnforcingConsecutive_5Xx = &wrappers.UInt32Value{Value: *p.EnforcingConsecutive5xx}
+	} else {
+		od.EnforcingConsecutive_5Xx = &wrappers.UInt32Value{Value: uint32(100)}
```

Comment:
This mean that we are setting default to 100, right? I thought that we wanted to leave default behaviour same which is not set the parameter.